extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -82,33 +87,33 @@ export class SubscriptionService {
         .then(() => this.processEntitlement_());
   }
 
-  /**
-   * @private
-   */
+  /** @private */
   processEntitlement_() {
     // TODO(@prateekbh): process and unblock marup here.
+    this.renderer_.toggleLoading(false);
```

Comment:
lets add a small test for this

Reply:
I'm actually going to remove this for now and leave it for you to add. I don't yet have a clear "phases" picture with the integration code.

Reply:
Dropped.